src/main/java/com/microsoft/lst_bench/client/JDBCConnection.java

```diff
@@ -95,4 +126,24 @@ public void close() throws ClientException {
       throw new ClientException(e);
     }
   }
+
+  private String createSQLWarningMessage(Statement s) throws ClientException {
```

Comment:
I have two comments about this method:
- It will still return a line `Warnings: ` even if there are no warnings. We should only print anything when there are warnings to avoid polluting the log files.
- It seems it simply concatenates the warning messages into a single String. I think we should log a warning per line rather than concatenating them, i.e., this method could return an array/list.

Reply:
Done!